relay-general/src/store/normalize.rs

```diff
@@ -254,6 +255,22 @@ impl<'a> NormalizeProcessor<'a> {
     }
 }
 
+fn normalize_app_start_spans(event: &mut Event) {
+    if let Some(spans) = event.spans.value_mut() {
+        for span in spans {
+            if let Some(span) = span.value_mut() {
+                if let Some(op) = span.op.value() {
+                    if op == "app_start_cold" {
+                        span.op.set_value(Some("app.start.cold".to_string()));
+                    } else if op == "app_start_warm" {
+                        span.op.set_value(Some("app.start.warm".to_string()));
+                    }
+                }
+            }
+        }
+    }
```

Comment:
This might be expensive, should we guard it only for the specific RN SDK version? Would that be ok?

Reply:
Sounds good. `semver` is already an indirect [dependency](https://github.com/getsentry/relay/blob/562b794c36d78a60874851fcb986ad137ba1baed/Cargo.lock#L3751-L3752) so you could use it to compare versions. Though what versions do we actually want to match on? Maybe a simple `startswith` or a regex would also work?

Reply:
I'll check, likely startsWith will do the job.